crates/astria-sequencer/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "astria-sequencer"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = "1"
```

Comment:
I was kind of forced to use anyhow to use the penumbra traits, eg. if I use eyre/color-eyre:
```
method `end_epoch` has an incompatible type for trait
expected signature `fn(&'life0 mut Arc<S>) -> Pin<Box<(dyn futures::Future<Output = Result<(), anyhow::Error>> + std::marker::Send + 'async_trait)>>`
   found signature `fn(&'life0 mut Arc<S>) -> Pin<Box<(dyn futures::Future<Output = Result<(), ErrReport>> + std::marker::Send + 'async_trait)>>`rustc[Click for full compiler diagnostic](rust-analyzer-diagnostics-view:/diagnostic%20message%20%5B0%5D?0#file%3A%2F%2F%2Fhome%2Fe%2Fastria%2Fcrates%2Fastria-sequencer%2Fsrc%2Faccounts%2Fcomponent.rs)
component.rs(41, 5): change the output type to match the trait: `Pin<Box<(dyn futures::Future<Output = Result<(), anyhow::Error>> + std::marker::Send + 'async_trait)>>`
```

this is in `impl Component for AccountsComponent` for example, but shows up other places. do you know a good way around this? I agree it would be better to be consistent

Reply:
No way around this. Very unfortunate that they hard coded this into their public interfaces. :(

Reply:
yeah :/ the alternative is that we don't use their `Component` trait and define our own, although I'm not sure if that'll break compatibility with them in the future. for now that seems fine though, we don't need to strictly follow their `Component` trait